res/CircleRecipes/RoPE_000/test.recipe

```diff
@@ -0,0 +1,73 @@
+    arg: "0.70711f" arg: "0.76041f" arg: "0.80902f" arg: "0.85264f" arg: "0.89101f"
+    arg: "0.92388f" arg: "0.95106f" arg: "0.97237f" arg: "0.98769f" arg: "0.99692f"
+    arg: "1.0f" arg: "0.99692f" arg: "0.98769f" arg: "0.97237f" arg: "0.95106f"
+    arg: "0.92388f" arg: "0.89101f" arg: "0.85264f" arg: "0.80902f" arg: "0.76041f"
+    arg: "0.70711f" arg: "0.64945f" arg: "0.58779f" arg: "0.5225f" arg: "0.45399f"
+    arg: "0.38268f" arg: "0.30902f" arg: "0.23345f" arg: "0.15643f" arg: "0.07846f"
+    arg: "0.0f" arg: "-0.07846f" arg: "-0.15643f" arg: "-0.23345f" arg: "-0.30902f"
+    arg: "-0.38268f" arg: "-0.45399f" arg: "-0.5225f" arg: "-0.58779f" arg: "-0.64945f"
+    arg: "-0.70711f" arg: "-0.76041f" arg: "-0.80902f" arg: "-0.85264f" arg: "-0.89101f"
+    arg: "-0.92388f" arg: "-0.95106f" arg: "-0.97237f" arg: "-0.98769f" arg: "-0.99692f"
+    arg: "-1.0f" arg: "-0.99692f" arg: "-0.98769f" arg: "-0.97237f" arg: "-0.95106f"
+    arg: "-0.92388f" arg: "-0.89101f" arg: "-0.85264f" arg: "-0.80902f" arg: "-0.76041f"
+    arg: "-0.70711f" arg: "-0.64945f" arg: "-0.58779f" arg: "-0.5225f" arg: "-0.45399f"
+    arg: "-0.38268f" arg: "-0.30902f" arg: "-0.23345f" arg: "-0.15643f" arg: "-0.07846f"    
+  }
+}
+operand {
+  name: "cos_table"
+  type: FLOAT32
+  shape { dim: 1 dim: 1 dim: 1 dim: 80 }
+  filler {
+    tag: "explicit"
+    arg: "1.0f" arg: "0.99692f" arg: "0.98769f" arg: "0.97237f" arg: "0.95106f"
+    arg: "0.92388f" arg: "0.89101f" arg: "0.85264f" arg: "0.80902f" arg: "0.76041f"
+    arg: "0.70711f" arg: "0.64945f" arg: "0.58779f" arg: "0.5225f" arg: "0.45399f"
+    arg: "0.38268f" arg: "0.30902f" arg: "0.23345f" arg: "0.15643f" arg: "0.07846f"
+    arg: "0.0f" arg: "-0.07846f" arg: "-0.15643f" arg: "-0.23345f" arg: "-0.30902f"
+    arg: "-0.38268f" arg: "-0.45399f" arg: "-0.5225f" arg: "-0.58779f" arg: "-0.64945f"
+    arg: "-0.70711f" arg: "-0.76041f" arg: "-0.80902f" arg: "-0.85264f" arg: "-0.89101f"
+    arg: "-0.92388f" arg: "-0.95106f" arg: "-0.97237f" arg: "-0.98769f" arg: "-0.99692f"
+    arg: "-1.0f" arg: "-0.99692f" arg: "-0.98769f" arg: "-0.97237f" arg: "-0.95106f"
+    arg: "-0.92388f" arg: "-0.89101f" arg: "-0.85264f" arg: "-0.80902f" arg: "-0.76041f"
+    arg: "-0.70711f" arg: "-0.64945f" arg: "-0.58779f" arg: "-0.5225f" arg: "-0.45399f"
+    arg: "-0.38268f" arg: "-0.30902f" arg: "-0.23345f" arg: "-0.15643f" arg: "-0.07846f"
+    arg: "-0.0f" arg: "0.07846f" arg: "0.15643f" arg: "0.23345f" arg: "0.30902f"
+    arg: "0.38268f" arg: "0.45399f" arg: "0.5225f" arg: "0.58779f" arg: "0.64945f"
+    arg: "0.70711f" arg: "0.76041f" arg: "0.80902f" arg: "0.85264f" arg: "0.89101f"
+    arg: "0.92388f" arg: "0.95106f" arg: "0.97237f" arg: "0.98769f" arg: "0.99692f"	    
+  }
+}
+operand {
+  name: "ofm"
+  type: FLOAT32
+  shape { dim: 1 dim: 1 dim: 1 dim: 80 }
+}
+operation {
+  type: "RoPE"
+  input: "ifm"
+  input: "sin_table"
+  input: "cos_table"
```

Comment:
I am not sure tables for `sin` and `cos` needs to be input of `RoPE`. It depends on implementation. If this recipe is for some specific implementation (e.g. `luci-interpreter`) and 2nd/3rd operands are optional, it may be okay.

Reply:
> I am not sure `sin`/`cos` tables are necessary. It depends on implementation. Some implementation may choose to call `sin` and `cos` directly w/o tables.
> 
> If 2nd/3rd operands are optional, it is okay.
thank you for your comments.
it is true that whether to use sin/cos table or sin/cos directly depends on implementation including app.
after the model is fixed, i will proceed in the direction of modifying it through performance comparison in implementation.